charts/jaeger/templates/_capabilities.tpl

```diff
@@ -0,0 +1,154 @@
+{{/* vim: set filetype=mustache: */}}
+
+{{/*
+Return the target Kubernetes version
+*/}}
```

Comment:
@DrFaust92 i dnt think we need these here. We already include common bitnami chart, which has all these defined: https://github.com/bitnami/charts/blob/main/bitnami/common/templates/_capabilities.tpl

Reply:
your change in hpa yaml files would just work (https://github.com/bitnami/charts/blob/main/bitnami/common/templates/_capabilities.tpl#L132)

Reply:
`{{- if semverCompare "<1.23-0" (include "common.capabilities.kubeVersion" .context) -}} ` doesnt work when i test locally. 
i think it needs to be 
`{{- if semverCompare "<1.23-0" (include "common.capabilities.kubeVersion" .) -}}` instead
otherwise this errors on:
```
Error: template: jaeger/templates/ingester-hpa.yaml:2:15: executing "jaeger/templates/ingester-hpa.yaml" at <include "common.capabilities.hpa.apiVersion" $>: error calling include: template: jaeger/charts/common/templates/_capabilities.tpl:133:7: executing "common.capabilities.hpa.apiVersion" at <semverCompare "<1.23-0" (include "common.capabilities.kubeVersion" .context)>: error calling semverCompare: Invalid Semantic Version

Use --debug flag to render out invalid YAML
```


i think i need to tweak the HPA resource a bit more, need to change `targetAverageUtilization` to `averageUtilization` with some conditional on api version

Reply:
nvm, found the issue and remove the capabilities file